src/components/Radio/index.tsx

```diff
@@ -0,0 +1,166 @@
+import styled from '@emotion/styled'
+import React, { forwardRef, InputHTMLAttributes } from 'react'
+import { color, device, fontSize, radius, space, transition } from '../..'
+import { Checkmark } from '../../icons'
+import { Label } from '../Label'
+
+const Container = styled.span`
+  position: relative;
+  display: inline-grid;
+  align-items: center;
+  justify-items: center;
+`
+
+const Icon = styled(Checkmark)`
+  grid-area: 1 / 1;
+  color: ${color.nova};
+  pointer-events: none;
+`
+
+const StyledInput = styled.input`
+  grid-area: 1 / 1;
+  background-color: ${color.nova};
+  height: 24px;
+  width: 24px;
```

Comment:
**question:** Should this use space instead? or is that only for margins/paddings?

Reply:
We could use `space` for that as well, we do that all the time so I don't see any problems with that.